src/peer_connection/mod.rs

```diff
@@ -1718,6 +1718,7 @@ impl RTCPeerConnection {
                             Arc::clone(&self.internal.dtls_transport),
                             Arc::clone(&self.internal.media_engine),
                             Arc::clone(&self.interceptor),
+                            !t.direction().has_send(),
```

Comment:
This is wrong, the act of adding a track should make the transceiver's direction become send. Thus this should be `true` unconditionally.

Reply:
alright

Reply:
alright